src/main/java/com/cognifide/SpringBootApplication.java

```diff
@@ -0,0 +1,36 @@
+package com.cognifide;
+
+import com.cognifide.domain.Book;
+import com.cognifide.repository.BookRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.context.annotation.Bean;
+
+/**
+ * Created by cognifide.nowak on 01.04.2016.
+ */
+@org.springframework.boot.autoconfigure.SpringBootApplication
```

Comment:
Probably you didn't want to use fully qualified name here (in this case `import` is welcomed). You should use fully qualified name only when the class name is ambiguous.